src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -163,42 +163,6 @@ def __getattribute__(self, name):
         return val
 
 
-def _get_ossrdbms_resource_id(cloud_name):
-    ossrdbms_mapper = {
```

Comment:
Is it because all these endpoints are defined in the `HARD_CODED_CLOUD_LIST` so that we can remove them ?

Reply:
Yes. The patch methods are used when the endpoint is in `HARD_CODED_CLOUD_LIST` but not in ARM Metadata yet.